crates/sp-domains-fraud-proof/src/storage_proof.rs

```diff
@@ -54,7 +55,7 @@ pub enum VerificationError {
 
 #[derive(Clone, Debug, Decode, Encode, Eq, PartialEq, TypeInfo)]
 pub enum FraudProofStorageKeyRequest<Number> {
-    BlockRandomness,
+    InvalidInherentExtrinsicData,
```

Comment:
Hang on, is this a breaking runtime API change?

This enum is passed to a runtime API here:
https://github.com/autonomys/subspace/blob/5675113160a5f969aa085f52a38fcc940ff4331d/crates/sp-domains-fraud-proof/src/lib.rs#L157-L166

If it is, I think we'll need to bump the runtime version in this PR as well.

Reply:
This should be fine as runtime API only fail when we actually call it and it has never been called yet as there is no domain initialized (while the host function change will crash the node upon runtime/client upgrade).

Reply:
Yep, in this case it would produce wrong results (because enum variants have changed indexes) or fail (because the expected data does not accompany the variant at that index). So that’s fine as long as most nodes are upgraded before then.

We might want to bump the runtime version for Taurus though.